test/fixtures/timdex/minimal_record.json

```diff
@@ -0,0 +1,5 @@
+{
+  "timdexRecordId": "minimal-record-456",
+  "title": "Minimal Test Record",
+  "source": ["Test Repository"]
```

Comment:
Is there a reason `source` is an array here? We only ever set one source per record in timdex... is the record spec in timdex that source is an array? If so... 🙈

Reply:
Oh lord. No, we do it correctly it in TIMDEX. This is just a hangover of working with Primo data, where everything is an array. 🙃